spec/features/comments_spec.rb

```diff
@@ -0,0 +1,35 @@
+require 'rails_helper'
+
+RSpec.feature "Comments", type: :feature do
+
+  scenario "create a comment" do
+    comment = FactoryBot.build(:comment)
+
+    visit root_path
+    click_link 'Read article'
```

Comment:
Rather than navigating from the main page, I'd go to the article page. It'll be something like `visit article_path(comment.article_id)`

Reply:
Fixed in commit d8658a9d8f9d8dd7e6a416bd13cec8bdbf25a750